src/Aspire.ProjectTemplates/templates/aspire-starter/8.2/Aspire-StarterApplication.1.Tests/Aspire-StarterApplication.1.Tests.csproj

```diff
@@ -17,11 +17,11 @@
     <PackageReference Include="Aspire.Hosting.Testing" Version="8.2.2" />
     <PackageReference Include="coverlet.collector" Version="6.0.2" Condition=" $(TestFramework) == 'xUnit.net' " />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.10.0" Condition=" $(TestFramework) != 'MSTest' " />
-    <PackageReference Include="MSTest" Version="3.4.3" Condition=" $(TestFramework) == 'MSTest' " />
+    <PackageReference Include="MSTest" Version="3.6.1" Condition=" $(TestFramework) == 'MSTest' " />
```

Comment:
This is the unsupported template (8.2), we don't update these.

Reply:
do we want to delete the 8.x templates yet?